rds_enhanced_monitoring/examples/generate_CWLogTestEvent.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+SCRIPTPATH=$(cd `dirname $0` && pwd)
+
+ts=$(($(date +%s%N)/100000))
```

Comment:
I think you want:

```suggestion
ts=$(($(date +%s%N)/1000000))
```

You get the timestamp in nanosecond so it would make sense to have it in millisecond, to send it in second once it's divided by 1000 in the lambda function

Reply:
also `date +%s%N` is not valid on macOS